test/ecdsatest.c

```diff
@@ -30,13 +30,14 @@ static size_t crv_len = 0;
 static EC_builtin_curve *curves = NULL;
 static OSSL_PROVIDER *fake_rand = NULL;
 
-static int fbytes(unsigned char *buf, size_t num)
+static int fbytes(unsigned char *buf, size_t num, ossl_unused const char *name,
```

Comment:
should you forward declare this as the typedef or will the compiler just pick this up?

Reply:
The compiler will pick this up when it is used later on.  Still, a forward declaration won't hurt.  I've added them.